packages/demo-app-ts/src/demos/topologyPackageDemo/OptionsViewBar.tsx

```diff
@@ -29,11 +28,11 @@ const OptionsContextBar: React.FC<{ controller: Controller }> = observer(({ cont
   };
 
   const layoutDropdown = (
-    <Split>
-      <SplitItem>
+    <Flex flexWrap={{ default: 'nowrap' }} gap={{ default: 'gapSm' }} alignItems={{ default: 'alignItemsCenter' }}>
+      <FlexItem>
```

Comment:
What if you don't have a flex layouts in the toolbar at all? 

you could use:
```
<ToolbarItem variant='label'>Layout:</ToolbarItem>
<ToolbarItem'><Dropdown.../></ToolbarItem>
```

Reply:
The reason I used a single `ToolbarItem` with a `Flex` for the label and input was so that the label and input would look more related (less space between them than between the input and the next label). Also, if the widow width is less, the new implementation cuts off the items rather than wrapping.

Before:
![image](https://github.com/user-attachments/assets/3e9d0b2e-06a5-46c7-bc8a-32047df671d0)

After:
![image](https://github.com/user-attachments/assets/36db8c35-518f-4ebe-993c-ea23cc933160)

Reply:
Thanks for clarifying @jeff-phillips-18 , reverted back to your changes in https://github.com/jenny-s51/react-topology/pull/11